song/routers/postRouter.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+const router = express.Router();
+
+const postController = require("../controllers/postController");
```

Comment:
```suggestion
const express = require("express");

const postController = require("../controllers/postController");

const router = express.Router();
```
코드 작성은 모듈을 불러오는 코드 먼저 항상 상단에 작성될 수 있게 하는것이 관례입니다.